app/trip.py

```diff
@@ -0,0 +1,87 @@
+from math import dist
+import datetime
+
+
+class Calculation:
+    def __init__(self, customer, shop):
+        # customer - instance of Customer
+        # shop - instance of Shop
+        self.fuel_price = customer.petrol_today
+        self.customer = customer
+        self.shop = shop
+
+    def get_distance(self) -> float:
+        return dist(self.customer.location, self.shop.location)
+
+    def cost_ride(self) -> float:
+        calculation_cost = self.customer.car / 100 * self.get_distance()
+        return calculation_cost * self.fuel_price
+
+    def cost_product(self) -> int:
+        total_cost = 0
+        for product in self.customer.products:
+            list_of_products = self.customer.products[product]

```

Comment:
maybe something like `product_amount` or `product_quantity` would be better? WDYT?

Reply:
yea i think you are right, i fix it.